lk2nd/device/dts/msm8909/msm8909-qrd-skub.dts

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8909 0>;
+	qcom,board-id = <QCOM_BOARD_ID_QRD 0x09>,
+			<QCOM_BOARD_ID(QRD, 1, 0) 0x09>;
+};
+
+&lk2nd {
+	lxf_p5001_b01 {
+		model = "Lenovo Yoga Tab 3 10 LTE";
```

Comment:
i proby should name it as lxf_p5001.

apparently there are other Lenovo Yoga Tab 3 10 (wifi/lte) are called lxf_p5001 but different sufixes like b01,j01,etc